tools/config/seed.config.ts

```diff
@@ -79,6 +79,14 @@ export class SeedConfig {
   APP_BASE = argv['base'] || '/';
 
   /**
+   * The flag for the lazy template loader option of the application.
+   * Per default the option is not set, but can be set by the `--lazy-template`
+   * flag when running `npm start`.
+   * @type {boolean}
+   */
+  LAZY_TEMPLATE = argv['lazy-template'];
```

Comment:
Lets call this option: `INLINE_TEMPLATES`. `LAZY_TEMPLATE` could be a bit misleading because of the lazy-loading feature of the router.